client/extensions/woocommerce/app/order/create.js

```diff
@@ -85,6 +86,7 @@ class Order extends Component {
 			dispatch( errorNotice( translate( 'Unable to create order.' ), { duration: 8000 } ) );
 		};
 
+		recordTrack( 'calypso_woocommerce_order_create' );
```

Comment:
it would be nice to move this as an incidental detail inside of `mapDispatchToProps()` so that the React component itself only needs to care about the fact that it can `saveOrder()` - will make it much simpler in testing to verify the component without having to try and worry about metrics tracking from test runs too!

Reply:
I'm not sure if we're set up to do this with the way we're [customizing/extending the `recordTrack` calls](https://github.com/Automattic/wp-calypso/tree/master/client/extensions/woocommerce/lib/analytics). I'm leaving this PR as-is but starting a discussion about how we can best do this going forward 🙂